src/Symfony/Component/Messenger/Exception/ValidationFailedException.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Exception;
+
+use Symfony\Component\Validator\ConstraintViolationListInterface;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class ValidationFailedException extends \RuntimeException implements ExceptionInterface
+{
+    protected $violations;
+    protected $violatingMessage;
```

Comment:
if no inheritance use case for these properties in core then they should be private

Reply:
Thank you